### DISCUSSION TITLE
No output for case "fv3_model_point_noah" (v6.0.0)

### DISCUSSION THREAD

**Discussion by wzzheng90** [NCAR/ccpp-scm#338] [Cases, Initial Conditions, Forcing]
Hi, I am trying to test v6.0.0 with the provided cases on HERA.  I got a problem for running case "fv3_model_point_noah" (with all default config).  
After running it:   ./run_scm.py -c fv3_model_point_noah -s SCM_GFS_v16 ,    I did not find the output file "output.nc" under ~/run/output_fv3_model_point_noah_SCM_GFS_v16.  There are only two files "fv3_model_point_noah_SCM_GFS_v16.nml" and "logfile" under this directory.   Also, I did not find any useful information from this "logfile".   Anything I did not set up correctly?    Thanks!    -- Weizhong

**Comment by grantfirl** [comment]
Hi Weizhong,

It looks like the input data that the physics expects has changed to the point that what the UFS_IC_generator.py script no longer is providing the right data. We will need to use the latest FV3GFS_io.F90 file in the fv3atm repository together with up-to-date UFS initial condition files to update the UFS_IC_generator.py script to regenerate a new fv3_model_point_noah case data file. Let me discuss with the DTC team about how/if we can accomplish this with our deliverables/funding.

-Grant

**Reply by wzzheng90** [reply]
Hi, Grant,

Thanks for your quick response.  Looking forward to the updated code/scripts.  
-Weizhong

**Comment by christianbsch** [comment]
Hello

I am having the same problem, no output file for the test case fv3_model_point_noah... there is any update about the FV3GFS_io.F90 and the UFS_IC_generator.py script? This latter only generates Initial conditions, but not forcing data... 

Thanks in advance for your help

Regards

Christian

**Reply by dustinswales** [reply]
Hi Christan,

Sorry you are having troubles.
The script in question has been updated. Please switch to the main branch and refer to the documentation

**Reply by christianbsch** [reply]
Hello Dustin, thanks for the reply. I have used the scripts from the main branch, however:
1. The fv3_model_point_noah experiment still doesn't generate any output
2. I tried to create again the fv3_model_point_noah netcdf file with UFS_IC_generator.py (with the main branch version), but it shows the following error: " UFS_IC_generator.py: error: the following arguments are required: -f/--forcing_dir ", 
Thank you

**Comment by ulmononian** [comment]
i was wondering if there was an update on this? 

on hera, i did the following

```
git clone --recursive -b v6.0.0 https://github.com/NCAR/ccpp-scm
cd ccpp-scm
source scm/etc/Hera_setup_intel.sh
./contrib/get_all_static_data.sh
./contrib/get_thompson_tables.sh
cd scm
mkdir bin; cd bin
cmake ../src 2>&1 | tee log.cmake.001
make -j8 2>&1 | tee log.make.001
./run_scm.py -m 2>&1 | tee log.run_scm.001
```

perhaps i misunderstood, but i expected all suites & cases to run with `./run_scm.py -m`, but there is no output folder created for `fv3_model_point_noah` generated in `ccpp-scm/scm/run`. there are output folders here for several others cases (e.g. arm, bomex, LASSO, astex) but not the ufs case. 

when i then try just `./run_scm.py -c fv3_model_point_noah -s SCM_GFS_v16`, i get the following error

```
(pyccpp) [Cameron.Book@hfe01 bin]$ ./run_scm.py -c fv3_model_point_noah -s SCM_GFS_v16
INFO: Setting up experiment fv3_model_point_noah with suite SCM_GFS_v16 using namelist input_GFS_v16.nml and tracers tracers_GFS_v16.txt
INFO: Parsing case configuration namelist /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/etc/case_config/fv3_model_point_noah.nml
INFO: Creating experiment configuration namelist input_experiment.nml in the run directory from /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/etc/case_config/fv3_model_point_noah.nml using SCM_GFS_v16 and input_GFS_v16.nml
INFO: Launching experiment fv3_model_point_noah with suite SCM_GFS_v16 using namelist input_GFS_v16.nml and tracers tracers_GFS_v16.txt
INFO: Passing control to "(cd /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run && time /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run/scm)"
CRITICAL: Execution of command "(cd /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run && time /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run/scm)" failed, exit code 127
    stdout: ""
    stderr: "/scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run/scm: error while loading shared libraries: libzstd.so.1: cannot open shared object file: No such file or directory

real	0m0.070s
user	0m0.001s
sys	0m0.015s"
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "./run_scm.py", line 733, in launch_executable
    (status, stdout, stderr) = execute(cmd, ignore_error = ignore_error)
  File "./run_scm.py", line 173, in execute
    raise Exception('Execution of command "{0}" failed, exit code {1}\n'.format(cmd, status))
Exception: Execution of command "(cd /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run && time /scratch1/NCEPDEV/stmp4/Cameron.Book/835/scm-v6/scm/etc/../../scm/run/scm)" failed, exit code 127
```

do i need to re-do the compile steps to invoke `./run_scm.py -c fv3_model_point_noah -s SCM_GFS_v16` or should this work from the initial build steps i shared at the top of this message?

any suggestions or insight would be most appreciated!

**Reply by wzzheng90** [reply]
@ulmononian  I got the similar error for this case with the latest SCM version.   The output folder was created for fv3_model_point_noah (generated in ccpp-scm/scm/run), but NO 'output.nc" was generated.   Looking forward to any suggestions/solutions too.  Thanks. 
-- Weizhong

**Reply by ulmononian** [reply]
i just repeated the instructions i shared in my first post, but using the ccpp-scm `master` branch. this time, `./run_scm.py -c fv3_model_point_noah -s SCM_GFS_v16` runs successfully. however, similar to what @wzzheng90 mentioned, an `output_fv3_model_point_noah_SCM_GFS_v16` was created in `run` but with no `output.nc` file (just a `.nml` and `logfile`).

any suggestions?

**Reply by hertneky** [reply]
We are currently looking into a solution and will respond as soon as we have something ready for you. Sorry for the inconvenience. Bottom line, the canned fv3_model_point_noah will not work and we are working on getting replay mode ready for you to generate the UFS forcing files instead. More to come soon.

**Reply by ulmononian** [reply]
@hertneky thank you very much for looking into this!

**Reply by wzzheng90** [reply]
@hertneky Any updates about this issue?  Thanks!
